upload_results.py

```diff
@@ -1,6 +1,7 @@
 import argparse
 import json
 import os
+from typing import Dict
```

Comment:
I prefer using imports `from __future__` because that will allow you to make something that works seamlessly with future versions of Python.

```suggestion
from __future__ import annotations
import argparse
import json
import os
```

Reply:
works for me